openvpn/factory.go

```diff
@@ -40,6 +40,7 @@ func NewClientConfig(
 	config.SetClientMode(remote, 1194)
 	config.SetTLSCACertificate(caCertPath)
 	config.SetTLSCrypt(tlsCryptKeyPath)
+	config.SetReconnectLimits()
```

Comment:
If number `2` were a parameter to this method, we could see what that limit is in here - also, that would allow us to change it in factory without changing config class itself.

Reply:
intuitively I want to pass parameters to a function that has a `set` prefix.
perhaps `RestrictConnectRetryMax(2)`

Reply:
Restrict prob better term here. Will rename.